### PR TITLE
Bump to OLM 1.12 metadata

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

This needs to wait until we have the 

```
registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.11.0:serverless-bundle
```

Image from CI 

/hold

### CODE REVIEW

File: hack/lib/catalogsource.bash [hunk 1]
```diff
@@ -81,7 +81,7 @@ spec:
         - -c
         - |-
           podman login -u $pull_user -p $token image-registry.openshift-image-registry.svc:5000 && \
-          /bin/opm registry add -d index.db --container-tool=podman --mode=replaces -b quay.io/openshift-knative/serverless-bundle:1.7.2,registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.8.0:serverless-bundle,registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.9.0:serverless-bundle,registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.10.0:serverless-bundle,image-registry.openshift-image-registry.svc:5000/$OLM_NAMESPACE/serverless-bundle && \
+          /bin/opm registry add -d index.db --container-tool=podman --mode=replaces -b quay.io/openshift-knative/serverless-bundle:1.7.2,registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.8.0:serverless-bundle,registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.9.0:serverless-bundle,registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.10.0:serverless-bundle,,registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.11.0:serverless-bundleimage-registry.openshift-image-registry.svc:5000/$OLM_NAMESPACE/serverless-bundle && \
```
Comment (markusthoemmes) [inline hack/lib/catalogsource.bash:84]:
```suggestion
          /bin/opm registry add -d index.db --container-tool=podman --mode=replaces -b quay.io/openshift-knative/serverless-bundle:1.7.2,registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.8.0:serverless-bundle,registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.9.0:serverless-bundle,registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.10.0:serverless-bundle,registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.11.0:serverless-bundle,image-registry.openshift-image-registry.svc:5000/$OLM_NAMESPACE/serverless-bundle && \
```

**Comment by matzew** [comment]
@warrenvw  the `registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.11.0:serverless-bundle` does that need some special build?   I do not see the image, yet

**Comment by markusthoemmes** [comment]
@matzew it's part of our CI builds.

**Comment by markusthoemmes** [comment]
Ha we missed it, fixing...

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Pull request merged**